activation/handler.go

```diff
@@ -545,6 +540,9 @@ func (h *Handler) handleAtx(ctx context.Context, expHash types.Hash32, peer p2p.
 	}
 	events.ReportNewActivation(vAtx)
 	h.log.WithContext(ctx).With().Info("new atx", log.Inline(vAtx), log.Int("size", len(msg)))
+
+	poetRoundEnd := h.clock.LayerToTime(atx.PublishEpoch.FirstLayer()).Add(h.poetCfg.PhaseShift - h.poetCfg.CycleGap)
+	metrics.ReportMessageLatency(pubsub.AtxProtocol, pubsub.AtxProtocol, time.Since(poetRoundEnd))
```

Comment:
Calculating latency till this moment in time will include validation time, which might increase as nipost validations are queued. Is it what we want to measure?
```suggestion
	metrics.ReportMessageLatency(pubsub.AtxProtocol, pubsub.AtxProtocol, receivedTime.Since(poetRoundEnd))
```

Reply:
actually i think it is better to remove this metric for atx, it is very hard to make sense of this metric in this case. because of the delays introduced by post verification, and because it can be gossiped at any point

Reply:
@poszu i removed atx metric, ptal

Reply:
I think the raised point applies to the other changes as well; is it intended to measure latency + validation time?

Reply:
Sorry for the late reply. Yes, the question still holds for the other metrics.

Reply:
yes it matters when message reached critical parts of the protocol, not so much when it was received.

beside that processing time is always added to propagation latency on every network hop, so if there are 3 hops - processing time will be added 3 times. so also for consistency i think it is better to measure total latency including processing time